web/packages/core/src/ruffle-player.ts

```diff
@@ -1295,12 +1311,14 @@ export class RufflePlayer extends HTMLElement {
                 break;
             default:
                 // Unknown error
-                errorBody = `
-                    <p>Ruffle has encountered a major issue whilst trying to display this Flash content.</p>
-                    <p>This isn't supposed to happen, so we'd really appreciate if you could file a bug!</p>
-                `;
+                errorBody = `<p>Ruffle has encountered a major issue whilst trying to display this Flash content.</p>`;
+                if (!isBuildOutdated) {
+                    errorBody += `<p>This isn't supposed to happen, so we'd really appreciate if you could file a bug!</p>`;
+                } else {
+                    errorBody += `<p>If you are the server administrator, please try to upload a more recent version of Ruffle.</p>`;
```

Comment:
As a server administrator, I would expect to know why should I update Ruffle. Let's try to explain this:
"If you are the server administrator, please try to upload a more recent version of Ruffle (current build is outdated: %BUILD_DATE%)."
Other wordings are of course welcomed.